### DISCUSSION TITLE
Can I use the same sandboxie supporter certificate with family and friends?

### DISCUSSION THREAD

**Discussion by LoserForever88** [sandboxie-plus/Sandboxie#3255] [Q&A Feedback]
As titled, would the supporter certificate (be it one year or perpetual) work that way on multiple devices?  If the answer is yes, is there a limit as to number of people I can share with?  Thank you.

**Comment by bastik-1001** [comment]
The [feature comparison table](https://sandboxie-plus.com/feature-comparison/) might be helpful. David might correct me on anything I say.

The personal licenses can be used on an unlimited number of your personal machines. (Personal covers all devices you use yourself. [...])

To the license with the name "Personal": The versions will keep their premium features, after the certificate expires, just that content that gets updated via Sandboxie-Live, like newer troubleshooting scripts won't be available to download. You basically buy the version plus updates for a year, plus support and keep the latest update with those features enabled, the support runs out.

There is a family pack which can be used on devices of personal use for you and your family.  (A family pack can be used for the entire family it is not required for the family to live in the same household, so the cert can be used for children which moved out or your grand parents.) Please note that when this certificate expires, not only does the support end, it also locks the premium features. Sandboxie still can be used, but the premium features are no longer available.

**Comment by DavidXanatos** [comment]
For family you need the family pack option your friends need to get their own certificates.
If you want to buy in bulk for many many friends we can arrange for a volume discount.
